src/firefly/js/visualize/projection/WavelengthHeaderParser.js

```diff
@@ -172,7 +172,7 @@ function isNonSeparableTABGroup(spectralCoords) {
     // all coordinates refer to the same table and output column
     const table = spectralCoords[0].tab.table;
     const ps3_1 = spectralCoords[0].tab.ps3_1.toUpperCase();
-    return spectralCoords.slice(1).every((c) => c.tab.table === table || c.tab.ps3_1.toUpperCase() === ps3_1);
+    return spectralCoords.slice().every((c) => c.tab.table === table && c.tab.ps3_1.toUpperCase() === ps3_1);
```

Comment:
I don't think you need the `slice` anymore since it is just doing an array copy for a boolean check.

Reply:
`spectralCoords.splice(1)` should remain, I’ve corrected that. We are considering groups of 2 or more spectral coordinates and comparing the tab parameters of the first coordinate with those of the remaining ones.